### PR TITLE
Add Distinct Meta Descriptions to Reported Pages

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

Adds meta descriptions to reported pages.

## Test Plan

Manual.

## Related PRs and Issues

N/A.

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes.

### CODE REVIEW

File: src/lib/utils/metadata.ts [hunk 1]
```diff
@@ -1,7 +1,18 @@
 import type { AuthorData, PostsData } from "$routes/blog/content";
 
 export const DEFAULT_HOST = 'https://appwrite.io';
-export const DEFAULT_DESCRIPTION = 'Appwrite is an open-source platform for building applications at any scale, using your preferred programming languages and tools.';
+export const DEFAULT_DESCRIPTION = 'Appwrite is an open-source backend platform offering essential APIs for building web and mobile applications.';
+export const ASSETS_DESCRIPTION = 'Resources for presenting the Appwrite brand, ensuring consistency in using our logos, colours, and other brand elements across various platforms and materials.'
```
Comment (TorstenDittmann) [inline src/lib/utils/metadata.ts:5]: we dont need consts for descriptions that are only used once, feel free to just move them to the places where they are used 👍🏻

**Review by TorstenDittmann — CHANGES_REQUESTED** [review]

File: src/lib/utils/metadata.ts (hunk 1 above)
Comment (ItzNotABug) [reply src/lib/utils/metadata.ts]: Done.

File: src/markdoc/layouts/Policy.svelte [hunk 2]
```diff
@@ -22,28 +22,38 @@
...
     let showToc = false;
 
+    // handle multiple policies
+    const pageId = $page.route.id;
+
+    if (pageId === '/cookies') {
+        seo.description = 'Read our cookie policy to understand your choices and manage your preferences.'
+    } else if (pageId === '/terms') {
+        seo.description = 'Review our Terms of Service to understand the rules and guidelines for using our open-source backend-as-a-service platform.'
+    }
```
Comment (TorstenDittmann) [inline src/markdoc/layouts/Policy.svelte:55]: not sure I like this, probably we need to have a better foundation to allow tying the description in the markdoc pages.

lets add an `export let description: string;` and add those descriptions in the `.markdoc` files 👍🏻

Good find that we didnt have this available in policy pages.

Comment (ItzNotABug) [reply src/markdoc/layouts/Policy.svelte:55]: Makes sense. Will do!

**Review by TorstenDittmann — APPROVED** [review]

**Comment by TorstenDittmann** [comment]
can you have @SnezhannaM take a look? :-)

**Comment by ItzNotABug** [comment]
> can you have @SnezhannaM take a look? :-)

The meta description were provided by @SnezhannaM. Does this require another look?

**Comment by TorstenDittmann** [comment]
> > can you have @SnezhannaM take a look? :-)
> 
> The meta descriptions were provided by @SnezhannaM. Does this require another look?

sweet, going to merge once ci is done :-)

**Pull request merged**